Apple/App/BurrowApp.swift

```diff
@@ -4,14 +4,50 @@ import SwiftUI
 @main
 @MainActor
 struct BurrowApp: App {
+    //To connect to the App Delegate
+    @NSApplicationDelegateAdaptor(AppDelegate.self) var delegate
```

Comment:
Would it be possible to fix all of the Swift lints? You should see them inside of Xcode